package.json

```diff
@@ -3,17 +3,27 @@
   "version": "0.9.2",
   "description": "",
   "dependencies": {
-    "assert": "^1.4.1",
     "bignumber": "^1.1.0",
     "bignumber.js": "^4.1.0",
     "moment": "^2.19.2",
-    "web3": "^0.20.2"
+    "shortid": "2.2.8",
+    "web3": "1.0.0-beta.26"
   },
   "devDependencies": {
-    "truffle": "^4.0.1",
-    "ganache-cli": "^6.0.3",
     "abi-decoder": "^1.0.9",
-    "ethereumjs-testrpc": "^6.0.3"
+    "eslint-config-google": "^0.9.1",
+    "eslint-config-standard": "^11.0.0-beta.0",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-node": "^5.2.1",
+    "eslint-plugin-promise": "^3.6.0",
+    "eslint-plugin-standard": "^3.0.1",
+    "ethereumjs-testrpc": "^6.0.3",
+    "ganache-cli": "^6.0.3",
+    "truffle": "^4.0.1",
+    "mustache": "2.3.0",
```

Comment:
Add mustache in dependencies else will break in npm while deployment

Reply:
Done